src/generate-types.ts

```diff
@@ -963,9 +963,9 @@ export function run(options: Options) {
               'enum',
               ts.createArrayLiteral(
                 schema.enum.map(i =>
-                  i === 'true'
```

Comment:
there is probably some murky reason why this was like so that escapes me. Could you write a test in codegen.spec.ts that the runtime allows `typeGuard.maker(true).succes()" but not `typeGuard.maker(false).success()`?

this might break existing code but the breakage seems to happen in codegen already instead of runtime so thats kind of ok

Reply:
@sakari done. If everything ok can you also describe how to deploy?